src/microbe_stage/Membrane.cs

```diff
@@ -411,52 +388,40 @@ private void InitializeMesh()
             }
         }
 
-        previousWorkBuffer.Capacity = vertices2D.Capacity;
-        nextWorkBuffer.Capacity = previousWorkBuffer.Capacity;
+        // Make the length longer to guarantee that everything fits easily inside the square
+        cellDimensions *= 100;
+
+        var startingBuffer = new List<Vector2>();
```

Comment:
To save on memory allocations, could you make this a field in this class? Like `private List<Vector2> previousWorkBuffer` used to be. And just clear it here before reusing it.